pom.xml

```diff
@@ -1155,6 +1146,56 @@
 				</plugins>
 			</build>
 		</profile>
+
+		<profile>
+			<id>dpkg</id>
```

Comment:
I would call this profile `deb`. `dpkg` is the tool that installs packages, not the tool that creates them.